src/App.js

```diff
@@ -11,9 +12,9 @@ function App() {
     <div className="App">
       <Routes>
         <Route path="/" element={<HomePage />} />
-        <Route path="/archived" element={<ARCHIVED />} />
-        <Route path="/trash" element={<TrashPage />} />
-        <Route path="/label" element={<LabelPage />} />
+        {/* <Route path="/archived" element={<ARCHIVED />} /> */}
+        {/* <Route path="/trash" element={<TrashPage />} /> */}
+        {/* <Route path="/label" element={<LabelPage />} /> */}
```

Comment:
I think you have commented out routes by mistake.

Reply:
I was refactoring and abstraction of my code, so  I have changed the file structure name.